docusaurus-docs/community/policies/backwards-compatibility.md

```diff
@@ -1,9 +1,280 @@
----
-description: Backwards compatibility policy
----
+# conda-store Backwards Compatibility Policy
 
-# Backwards compatibility policy
+## Introduction
+
+In software development, it is essential to strike a balance between progress
+and maintaining a stable environment for existing users. This policy guides how
+the conda-store project will handle changes to software and services, ensuring
+that they do not disrupt the workflows of current users, while still enabling
+innovation and forward progress.
+
+## Breaking versus non-breaking changes
+
+Breaking code changes refer to modifications or updates made to software that
+have the potential to disrupt the functionality of existing applications,
+integrations, or systems. Breaking changes involve:
+
+- removing existing functionality
+- altering existing functionality
+- adding new requirements such as making a previously optional parameter required.
+
+These changes can lead to compatibility issues, causing frustration for
+end-users, higher maintenance costs, and even system downtime, thus undermining
+the trust and reputation of the software or service provider. Changes are only
+breaking if they impact users through the REST API, the Python API, or the
+Database.
+
+In contrast, non-breaking changes can add functionality or reduce requirements
+such as making a previously required parameter optional.
+These changes allow software to evolve and grow without negatively impacting
+existing users.
+
+:::info[TLD;R]
+
+conda-store users should be able to upgrade to newer versions without worrying
+about breaking existing integrations. Newer features will be available for users
+to adopt when needed, while all existing code should continue to work.
 
-:::note
-This page is in active development, and will be available soon.
 :::
+
+## Specific implementation guidance
+
+### Database changes
+
+Databases are one of the most critical areas to ensure there are no breaking
+changes. Databases hold the state for the application.  Introducing breaking
+changes to the database can be destructive to data and prevent rolling back to
+earlier versions of conda-store. To maintain backward compatibility we follow
+these principles:
+
+- New columns or tables should be added instead of removing or altering existing
+ones.  - Columns and tables should not be renamed. Aliases should be used for
+poorly named existing columns or tables.
```

Comment:
```suggestion
- New columns or tables should be added instead of removing or altering existing
  ones. 
- Columns and tables should not be renamed. Aliases should be used for
  poorly named existing columns or tables.
```